smtchecker.ts

```diff
@@ -25,7 +25,7 @@ function handleSMTQueries (inputJSON, outputJSON, solver) {
   return inputJSON;
 }
 
-function smtCallback (solver) {
+export function smtCallback (solver) {
```

Comment:
Does this needs to be exported?

Reply:
nope, just default exported, this has been removed.